op.go

```diff
@@ -96,12 +96,6 @@ type BinaryOp interface {
 	IsBinary() bool
 }
 
-type BestDoer interface {
```

Comment:
Already discussed: Did not find any reference anywhere, so I consider it as dead code